api/lib/domain/usecases/create-sessions.js

```diff
@@ -6,18 +6,23 @@ module.exports = async function createSessions({ data }) {
   if (data.length === 0) {
     throw new UnprocessableEntityError('No data in table');
   }
+
+  const sessionsToValidate = data.map((data) => {
+    return {
+      certificationCenterId: '',
+      certifcenter: '',
+      accessCode: '',
+      address: data['* Nom du site'],
+      room: data['* Nom de la salle'],
+      date: data['* Date de début'],
+      time: data['* Heure de début (heure locale)'],
+      examiner: data['* Surveillant(s)'],
+      description: data['Observations (optionnel)'],
+    };
```

Comment:
On peut eviter de boucler 2 fois sur toutes les sessions:
```suggestion
    const sessionToValidate = {
      certificationCenterId: '',
      certifcenter: '',
      accessCode: '',
      address: data['* Nom du site'],
      room: data['* Nom de la salle'],
      date: data['* Date de début'],
      time: data['* Heure de début (heure locale)'],
      examiner: data['* Surveillant(s)'],
      description: data['Observations (optionnel)'],
    };
     sessionValidator.validate(sessionToValidate);
     return sessionToValidate;
 }
```

Reply:
On ne map déjà qu'une fois sur l'ensemble des sessions, ou alors j'ai pas compris ta remarque 🤔